Source/WebCore/accessibility/AXObjectCache.cpp

```diff
@@ -1304,6 +1304,13 @@ void AXObjectCache::handleChildrenChanged(AccessibilityObject& object)
     else if (auto* axRow = dynamicDowncast<AccessibilityTableRow>(object)) {
         if (auto* parentTable = axRow->parentTable())
             deferRecomputeTableCellSlots(*parentTable);
+    } else if (auto* scrollView = dynamicDowncast<AccessibilityScrollView>(object)) {
+        // When we get a childrenChanged for a iframe (a ScrollView that is not the root web area), then we need to
+        // dirty the web area's subtree since the scroll area doesn't have a node nor renderer
```

Comment:
It could be nice to explain why not having a node or renderer matters, e.g. "doesn't have a node or renderer, thus failing the check below and returning early."

Reply:
I like this, will add to this comment